test/functional/sqlsrv/MsHelper.inc

```diff
@@ -370,32 +425,106 @@ function insertRow($conn, $tbname, $inputs, &$r = null, $api = INSERT_QUERY)
         $params = array();
         foreach ($inputs as $key => $input) {
             if (is_object($input)) {
-                array_push($params, $input->bindParamArr($inputs[$key]));               
+                array_push($params, $input->bindParamArr($input->value));
```

Comment:
this is very strange looking. If you're calling a member function from $input, the function will have access to everything in $input. Passing in $input->value is not necessary. Please change bindParamArr function accordingly.

Reply:
I reverted the change now. Thanks!